docusaurus/docs/guides/kubernetes/hosting/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
-  "label": "Hosting OpenZiti",
+  "label": "Self Hosting",

```

Comment:
Isn't any "hosting" by definition "self"? Feels like it to me. I prefer the former wording

Reply:
Yeah, I don't like "Self Hosting" either. I was trying to distinguish between the two types of info we have about K8s: "deployments" and "tunneling" without further overloading the term "hosting." I'll try those as section labels.